tests/FixerFactoryTest.php

```diff
@@ -530,6 +531,60 @@ public function testNoDoubleConflictReporting()
         );
     }
 
+    public function testFixerEndPoints()
```

Comment:
test should be atomic, check one fixer and use data provider to provide all fixers step by step